### DISCUSSION TITLE
CONFUSED...HELP!!!!!

### DISCUSSION THREAD

**Discussion by ghost** [Repl-it-Coders/Repl.it-2020#5] [Q&A] [answered]
@whippingdot 
**Anyone else confused**
1) didn't we already finish repl.it 2020
2) can someone explain what the heck is going on 
3) what should I be working on?

**Comment by whippingdot** [comment, marked as answer]
@ColePete 
1. Yes
2. We are adding more changes to the project as things were deleted as we couldn't finish them. We are finishing things we deleted in the shared repl
3. Umm...You can either help with making the suggestions be sent to a file/DB/JSON file or you can help by remaking the whole project so that the home page is the home.html and the link for that is just / not /home or you can help by putting repl auth in the suggestions page.

Explanation to 3: We are putting the auth on the suggestions page underneath repl.it username instead of having a full page for it as we don't need anyone to sign in unless we need their account username. That is why we need to put home.html in / instead of /home.
